feather/server/src/connection_worker.rs

```diff
@@ -145,6 +145,7 @@ impl Worker {
         tokio::task::spawn(async move {
             let result = reader.race(writer).await.expect("task panicked");
             if let Err(e) = result {
+                log::trace!("Panic error is: {}", e);
```

Comment:
There is no need in this line, `disconnect_message(e)` returns `Debug` error message of the error, which is almost always more useful than `Display`. The exception is `"disconnected"` for `ErrorKind::UnexpectedEof`, but that's more than enough to understand that the player has left the server.

Reply:
I agree, the error is already logged in line 150, this is superfluous

Reply:
This was an accidental change I made while I was exploring the repo. I didn't think new changes in my fork were incorporated. I'll be more careful next time.

Reply:
Ah I see. Then please revert the commit.

Reply:
That was done.

Reply:
(I am the same person as @csm-yujinkim) one is my school account and this is my main. Sorry for the switchup but it's because I'm graduating soon.